packages/toolkit/src/view/knowledge/components/CreateKnowledgeBaseCard.tsx

```diff
@@ -196,24 +121,36 @@ Text Chunks #: ${totalChunks}
     await onUpdateKnowledgeBase(data, knowledgeBase.catalogId);
     setEditDialogIsOpen(false);
   };
+
   const handleMouseMove = (e: React.MouseEvent) => {
-    const rect = e.currentTarget.getBoundingClientRect();
-    setMousePosition({
-      x: e.clientX - rect.left - 300,
-      y: e.clientY - rect.top - 150,
-    });
+    if (cardRef.current) {
+      const rect = cardRef.current.getBoundingClientRect();
+      setMousePosition({
+        x: e.clientX - rect.left,
+        y: e.clientY - rect.top,
+      });
+    }
+  };
+
+  const handleMouseEnter = () => {
+    setIsHovered(true);
+  };
+
+  const handleMouseLeave = () => {
+    setIsHovered(false);
   };
 
   return (
     <React.Fragment>
       <Tooltip.Provider>
-        <Tooltip.Root disableHoverableContent={isMenuOpen}>
+        <Tooltip.Root open={isHovered && !isMenuOpen}>
           <Tooltip.Trigger asChild>
             <div
+              ref={cardRef}
               className="flex h-[175px] w-[360px] cursor-pointer flex-col rounded-md border border-semantic-bg-line bg-semantic-bg-primary p-5 shadow hover:bg-semantic-bg-base-bg"
               onClick={onCardClick}
-              onMouseEnter={() => setIsHovered(true)}
-              onMouseLeave={() => setIsHovered(false)}
+              onMouseEnter={handleMouseEnter}
+              onMouseLeave={handleMouseLeave}
               onMouseMove={handleMouseMove}
```

Comment:
@thewbuk What is the outcome you want to achieve with the tooltip that you can not achieve with its native behavior?